CorsixTH/Lua/dialogs/bottom_panel.lua

```diff
@@ -721,7 +721,7 @@ function UIBottomPanel:addDialog(dialog_class, extra_function)
         self.ui:setEditRoom(false)
         local dialog = _G[dialog_class](self.ui)
         if extra_function then
-          _G[dialog_class][extra_function](d)
+          _G[dialog_class][extra_function](dialog)
```

Comment:
Incidentally the only place this is used is to call ```UIBankManager.showStatistics``` https://github.com/CorsixTH/CorsixTH/blob/master/CorsixTH/Lua/dialogs/fullscreen/bank_manager.lua#L318

The argument to that function is a bool for whether to replace the cursor.